### ISSUE TITLE
[FEATURE] Support for package manager: antigen

### ISSUE THREAD

**Issue by jayvdb** [tversteeg/emplace#171]
**What is the name of the package manager you would like to see support for?**
[antigen](https://github.com/zsh-users/antigen)

**What operating systems or derivative does this package manager support?**
- [x] Windows
- [x] Mac
- [x] Linux
- [x] BSD
- [ ] Redox
- [ ] iOS
- [ ] Android
- [ ] Other, namely:

**If Linux, is it only for a specific distro?**
- [ ] Yes, namely:
- [x] No

**What type of package manager is this?**
- [ ] Programming language specific (e.g. Cargo for Rust)
- [ ] General purpose (e.g. APT on Debian)
- [x] Other, namely: zsh plugins

**Are administrator priviliges required? (sudo, etc.)**
- [ ] Yes
- [x] No

**Comment by tversteeg** [comment]
Can this be used from the command line to install software, like `antigen bundle package`? Because I've skimmed the documentation a bit, and it looks like bundles are defined in a configuration file.

**Comment by jayvdb** [comment]
Yes it can be run on the cli, and I usually do that first, to ensure that it installs and activates correctly before putting it into the configuration file.

**Comment by tversteeg** [comment]
But does Emplace make sense in that workflow? It seems just syncing the configuration file in the same repository already reaches that goal?

**Comment by jayvdb** [comment]
antigen plugins can have significant dependencies which would be tracked in emplace.  Having the antigen plugin also captured helps keep a temporal log of what is being installed and why.
